src/AWSS3.jl

```diff
@@ -745,34 +745,13 @@ function s3_multipart_upload(aws::AWSConfig, bucket, path, io::IOStream,
 end
 
 
+using DataStructures
 using MbedTLS
 
 
-"""
-    s3_sign_url([::AWSConfig], bucket, path, [seconds=3600];
-                [verb="GET"], [content_type="application/octet-stream"])
-
-Create a
-[pre-signed url](http://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html) for `bucket` and `path` (expires after for `seconds`).
-
-To create an upload URL use `verb="PUT"` and set `content_type` to match
-the type used in the `Content-Type` header of the PUT request.
-
-```
-url = s3_sign_url("my_bucket", "my_file.txt"; verb="PUT")
-Requests.put(URI(url), "Hello!")
-```
-```
-url = s3_sign_url("my_bucket", "my_file.txt";
-                  verb="PUT", content_type="text/plain")
-
-Requests.put(URI(url), "Hello!";
-             headers=Dict("Content-Type" => "text/plain"))
-```
-"""
-function s3_sign_url(aws::AWSConfig, bucket, path, seconds=3600;
-                     verb="GET", content_type="application/octet-stream",
-                     protocol="http")
+function _s3_sign_url_v2(aws::AWSConfig, bucket, path, seconds=3600;
+                         verb="GET", content_type="application/octet-stream",
+                         protocol="http")
```

Comment:
Although this package doesn't officially support [BlueStyle](https://github.com/invenia/BlueStyle) I would like to move towards it.

```suggestion
function _s3_sign_url_v2(
    aws::AWSConfig, bucket, path, seconds=3600;
    verb="GET", content_type="application/octet-stream", protocol="http",
)
```

Reply:
No problems! I was just trying to match the style of the rest of the file. I can go ahead and use BlueStyle